common-model/src/main/java/com/sequenceiq/common/api/backup/model/BackupCloudwatchParams.java

```diff
@@ -0,0 +1,86 @@
+package com.sequenceiq.common.api.backup.model;
+
+import java.io.Serializable;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.sequenceiq.common.api.backup.doc.BackupModelDescription;
+import com.sequenceiq.common.api.telemetry.model.CloudwatchParams;
+import com.sequenceiq.common.api.telemetry.model.CloudwatchStreamKey;
+
+import io.swagger.annotations.ApiModelProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class BackupCloudwatchParams implements Serializable {
+
+    @ApiModelProperty
+    @NotNull
+    private String instanceProfile;
+
+    @ApiModelProperty(BackupModelDescription.CLOUDWATCH_PARAMS_REGION)
+    private String region;
+
+    @ApiModelProperty(BackupModelDescription.CLOUDWATCH_PARAMS)
+    private CloudwatchStreamKey streamKey = CloudwatchStreamKey.HOSTNAME;
+
+    public String getInstanceProfile() {
+        return instanceProfile;
+    }
+
+    public void setInstanceProfile(String instanceProfile) {
+        this.instanceProfile = instanceProfile;
+    }
+
+    public CloudwatchStreamKey getStreamKey() {
+        return streamKey;
+    }
+
+    public void setStreamKey(CloudwatchStreamKey streamKey) {
+        this.streamKey = streamKey;
+    }
+
+    public String getRegion() {
+        return region;
+    }
+
+    public void setRegion(String region) {
+        this.region = region;
+    }
+
+    @JsonIgnore
+    public static BackupCloudwatchParams copy(BackupCloudwatchParams cloudwatchParams) {
```

Comment:
This is a copy of the current working logging code. 

In TelemetryConverter.java, in the createLoggingRequestFromSource() method, there is one line as below,
`loggingRequest.setCloudwatch(CloudwatchParams.copy(logging.getCloudwatch()));`

It creates a deep copy of the cloudwatch variable and assigns it to the logging request so that the logging object in telemetry and logging object in LoggingRequest are decoupled. Change one will not change the other one. 

When I separated the backup from logging, I did the same thing to create a deep copy so that the variable is decoupled in different objects.